addons/im_livechat/static/tests/embed/livechat_service.test.js

```diff
@@ -108,31 +112,33 @@ test("Only necessary requests are made when creating a new chat", async () => {
     await triggerHotkey("Enter");
     await contains(".o-mail-Message", { text: "Hello!" });
     const [threadId] = pyEnv["discuss.channel"].search([], { order: "id DESC" });
-    await waitForSteps([
-        `/im_livechat/get_session - ${JSON.stringify({
-            channel_id: livechatChannelId,
-            anonymous_name: "Visitor",
-            previous_operator_id: operatorPartnerId,
-            persisted: true,
-        })}`,
-        `/mail/message/post - ${JSON.stringify({
-            post_data: {
-                body: "Hello!",
-                email_add_signature: true,
-                message_type: "comment",
-                subtype_xmlid: "mail.mt_comment",
-            },
-            thread_id: threadId,
-            thread_model: "discuss.channel",
-            context: {
-                lang: "en",
-                tz: "taht",
-                uid: serverState.userId,
-                allowed_company_ids: [1],
-                temporary_id: 0.8200000000000001,
-            },
-        })}`,
-    ]);
+    await waitStoreFetch(
+        [
+            "failures", // called because mail/core/web is loaded in test bundle
+            "systray_get_activities", // called because mail/core/web is loaded in test bundle
+            "init_messaging",
+        ],
```

Comment:
Note: there are 2 init messaging calls.

This was hidden because of lazy session. A follow up PR should remove it.